libs/sdk-ui-kit/src/Header/tests/generateHeaderMenuItemsGroups.test.ts

```diff
@@ -165,9 +306,30 @@ describe("generateHeaderMenuItemsGroups", () => {
         ]);
     });
 
+    it("should not return manage item with workspace in uri if canManageMetric is false and enableRenamingProjectToWorkspace is true", () => {
```

Comment:
Please add one more test case when `enableRenamingProjectToWorkspace` is `false`.

`it("should not return manage item with project in uri if canManageMetric is false and enableRenamingProjectToWorkspace is false"`

Reply:
As the existing test can be considered for case when `enableRenamingProjectToWorkspace` is `false`, the comment is marked as resolved.